test/imperative.js

```diff
@@ -192,3 +192,17 @@ metatests.testSync('testAsync must return a test', (test) => {
   test.ok(t);
   t.end();
 });
+
+metatests.test('Must catch unhandledExeptions', (test) => {
+  const t = new metatests.ImperativeTest('Throwing test', () => {
+    throw new Error();
+  }, { async: false });
+
+  setTimeout(() => {
+    test.assert(t.done, 'Must finish');
```

Comment:
I don't think that message should be capitalized here and below, to be consistent with the other tests' messages.